src/core/prototypes/guides/polar_coordinator.ts

```diff
@@ -0,0 +1,712 @@
+              return element._id === elementID;
+            });
+            if (found) {
+              const elementState = found[1];
+              elementState.attributes[attribute] = value;
+            } else {
+              for (const [element, elementState] of zipArray(
+                manager.chart.elements,
+                manager.chartState.elements
+              )) {
+                if (Prototypes.isType(element.classID, "plot-segment")) {
+                  const plotSegment = element as Specification.PlotSegment;
+                  const plotSegmentState = elementState as Specification.PlotSegmentState;
+                  for (const glyphState of plotSegmentState.glyphs) {
+                    const glyph = Prototypes.findObjectById(
+                      manager.chart,
+                      plotSegment.glyph
+                    ) as Specification.Glyph;
+                    const found = zipArray(glyph.marks, glyphState.marks).find(
+                      ([element, elementState]) => {
+                        return element._id === elementID;
+                      }
+                    );
+
+                    if (found) {
+                      const elementState = found[1];
+                      elementState.attributes[attribute] = value;
+                    }
+                  }
+                }
+              }
+            }
+          },
+          manager
+        )
+      );
+    }
+  }
+
+  public getValueNamesForAngular(): string[] {
+    const attrs = [];
+    for (let i = 0; i < this.object.properties.angularGuidesCount; i++) {
+      const name = `angularValue${i}`;
+      attrs.push(name);
+      if (this.state) {
+        if (this.state.attributes[name] == null) {
+          this.state.attributes[name] = 0;
+        }
+      }
+    }
+    return attrs;
+  }
+
+  public getValueNamesForRadial(): string[] {
+    const attrs = [];
+    for (let i = 0; i < this.object.properties.radialGuidesCount; i++) {
+      const name = `radialValue${i}`;
+      attrs.push(name);
+      if (this.state) {
+        if (this.state.attributes[name] == null) {
+          this.state.attributes[name] = 0;
+        }
+      }
+    }
+    return attrs;
+  }
+
+  public getValueNamesForPoints(): string[] {
+    const attrs = [];
+    for (let i = 0; i < this.object.properties.angularGuidesCount; i++) {
+      for (let j = 0; j < this.object.properties.radialGuidesCount; j++) {
+        const nameX = `point${i}${j}X`;
+        attrs.push(nameX);
+        if (this.state) {
+          if (this.state.attributes[nameX] == null) {
+            this.state.attributes[nameX] = 0;
+          }
+        }
+        const nameY = `point${i}${j}Y`;
+        attrs.push(nameY);
+        if (this.state) {
+          if (this.state.attributes[nameX] == null) {
+            this.state.attributes[nameX] = 0;
+          }
+        }
+      }
+    }
+    return attrs;
+  }
+
+  public get attributeNames(): string[] {
```

Comment:
Can you make an enum of all these strings. See

https://github.com/microsoft/charticulator/blob/fd881377ffba35d24df3b0dfcba88208988fd1a4/src/core/prototypes/guides/guide.ts#L22

https://github.com/microsoft/charticulator/blob/fd881377ffba35d24df3b0dfcba88208988fd1a4/src/core/prototypes/guides/guide.ts#L62

Since this is a new file, I'd like to use less strings everywhere.